Globally say "Microsoft Edge Add-ons", without "store", "site", or "website" (#3409)
MicrosoftDocs/edge-developer: microsoft-edge/extensions-chromium/developer-guide/best-practices.md

---
# Best practices for extensions

> Follow these Best Practices to implement your extension and list it on the Microsoft Edge Add-ons.

Reviewer: ```suggestion
Follow these Best Practices to implement your extension and list it at [Microsoft Edge Add-ons](todo: url for site).
```
